src/main/java/frc/robot/subsystems/ShooterHoodSubsystem.java

```diff
@@ -55,29 +55,23 @@ public ShooterHoodSubsystem() {
     hoodPID.setSmartMotionAllowedClosedLoopError(allowedErr, 0);
 
     hoodNEO.getForwardLimitSwitch(CANDigitalInput.LimitSwitchPolarity.kNormallyOpen).enableLimitSwitch(false);
-    hoodLimitSwitch.enableLimitSwitch(enableLimit);
 
-    hoodNEO.setIdleMode(IdleMode.kCoast);
+    hoodNEO.setIdleMode(IdleMode.kBrake);
   }
 
   public void moveHood(double posRequest) {
     // Used for autonomous and vision-tied control of the shooter hood.
     hoodPID.setReference(posRequest, ControlType.kSmartMotion, 0);
-    enableLimit = false;
   }
 
   public void deployHood() {
-    while (hoodPos < maxHoodPos) {
-      hoodPID.setReference(-0.1, ControlType.kSmartMotion, 0);
-    }
-    isDeployed = true;
+    deploy = true;
+    stow = false;
```

Comment:
If deploy and stow are always opposites (which they are), they should never be represented by two different variables.

Options are:

- Single boolean called either stow OR deploy
- Enum indicating different state options, of which there are currently 2.